go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	github.com/rs/cors v1.11.0
 	github.com/santhosh-tekuri/jsonschema/v5 v5.3.1
 	github.com/seehuhn/mt19937 v1.0.0
+	github.com/slok/go-http-metrics v0.12.0
```

Comment:
I'm not a big fan of introducing single-use dependencies. don't we already have something in our dependency list that gives us the functionality the new package would provide?